### PR TITLE
Share contact infos with Crossref

### PR DESCRIPTION
**Pull request by njahn82**
Following #144 this stores the email address in `.Renviron` and shares it with Crossref via the user agent

### CODE REVIEW

**Comment by njahn82** [comment]
Sorry @sckott I messed it up again. Now tests are not failing because of missing email info in `.Renviron`

File: R/zzz.R [hunk 1]
```diff
@@ -16,14 +16,17 @@ rcrossref_ua <- function() {
...
   paste0(versions, collapse = " ")
 }
 
-cr_GET <- function(endpoint, args, todf = TRUE, on_error = warning, parse = TRUE, ...) {
+cr_GET <- function(endpoint, args, todf = TRUE, on_error = warning, parse = TRUE, 
+                   ...) {
   url <- sprintf("https://api.crossref.org/%s", endpoint)
   cli <- crul::HttpClient$new(
     url = url,
+    opts = list(verbose = TRUE),
```
Comment (sckott) [inline R/zzz.R]: @njahn82 did you mean to leave this here?

**Comment by sckott** [comment]
thanks for this, a comment above. 

also, i think in your original PR you had added some tests. can you do that?

**Comment by njahn82** [comment]
Sorry for the delay, I added some tests, but not for all functions to keep run time of tests low. Hope I did not miss something

**Comment by sckott** [comment]
LGTM @njahn82 

thanks much for doing this work!

**Pull request merged**